lib/ash/resource.ex

```diff
@@ -44,22 +44,26 @@ defmodule Ash.Resource do
                         )
 
   @moduledoc """
-  The entry point for creating an `Ash.Resource`.
+  A resource is a public, standardized API on top of your data layer(s). A resource defines which CRUD operations are allowed to be performed, which data attributes are exposed, as well as relationships to other resources.
```

Comment:
You can technically have a resource w/ no data layer, and just use them to describe resources. A resource is a static definition of an entity in your system. Think "Posts" or "Authors" or "Tweets". They declare their CRUD operations, attributes, relationships, and other behavior, all of which can be customized to map to your underlying business logic. Resources can use "Data Layers", which allow them to be persisted and manipulated via `Ash`. Most importantly, front ends can take your resources and scaffold entire APIs/interaction engines based entirely off of that static declaration."